docs/pages/router/basics/layout.mdx

```diff
@@ -0,0 +1,166 @@
+
+<FileTree
+  files={[
+    ['app/(tabs)/\_layout.tsx'],
+    ['app/(tabs)/index.tsx'],
+    ['app/(tabs)/feed.tsx'],
+    ['app/(tabs)/profile.tsx'],
+  ]}
+/>
+
+In the **\_layout.tsx** file, return a `Tabs` component:
+
+```tsx _layout.tsx
+import { Tabs } from 'expo-router';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tabs.Screen
+        name="index"
+        options={{
+          title: 'Home',
+          tabBarIcon: ({ color }) => <MaterialIcons size={28} name="house.fill" color={color} />,
+        }}
+      />
+      <!-- Add more tabs here -->
+    </Tabs>
+  );
+}
+```
+
+This will cause the **index.tsx**, **feed.tsx**, and **profile.tsx** files to appear together in the same bottom tabs navigator. This `Tabs` component uses [React Navigation's native bottom tabs](https://reactnavigation.org/docs/bottom-tab-navigator/) and supports the same options.
+
+In the case of `Tabs`, you will likely want to define the tabs in the navigator, as this influences the order in which tabs appear, the title, and the icon inside the tab. The index route will be the default selected tab.
+
+## Slot
+
+In some cases, you may want a layout without a navigator. This is helpful for adding a header or footer around the current route, or for displaying a modal over any route inside a directory. In this case, you can use the `Slot` component, which serves as a placeholder for the current child route.
+
+Consider the following file structure:
+
+<FileTree
+  files={[
+    ['app/social/\_layout.tsx'],
+    ['app/social/index.tsx'],
+    ['app/social/feed.tsx'],
+    ['app/social/profile.tsx'],
+  ]}
+/>
+
+For example, you may want to wrap any route inside the **social** directory with a header and footer, but you want navigating between the pages to simply replace the current page rather than pushing new pages onto a stack, which can then later be popped off with a "back" navigation action. In the **\_layout.tsx** file, return a `Slot` component surrounded by your header and footer:
+
+```tsx app/social/_layout.tsx
+import { Slot } from 'expo-router';
+
+export default function Layout() {
+  return (
+    <>
+      <Header />
+      <Slot />
+      <Footer />
+    </>
+  );
+}
+```
+
+## Other layouts
+
+These are just a few examples of common layouts to give you an idea of how it works. There's much more you can do with layout:
+
+- Implement a [Drawer navigator](/router/advanced/drawer)
+- Replace the default tabs with [fully customized tabs](/router/advanced/custom-tabs)
+- Use a [modal](/router/advanced/modals) to display a page with transparency, such that the parent navigator is still visible underneath.
+- [Adapt any navigator that is compatible with React Navigation](/versions/latest/sdk/router/#withlayoutcontextnav-processor), including top tabs, bottom sheets, and more.
```

Comment:
```suggestion
- Use a [modal](/router/advanced/modals) to display a page with transparency, such that the parent navigator is still visible underneath
- [Adapt any navigator that is compatible with React Navigation](/versions/latest/sdk/router/#withlayoutcontextnav-processor), including top tabs, bottom sheets, and more
```

Let's remove the period at the end of the sentences to be consistent with other list items.